### PR TITLE
DB: Fixing several Npc issues

### PR DESCRIPTION
**Pull request by poszer**
- NPC Emerald Skytalon now correctly attack target.
- The players can now interact with NPC Geirrvif.
- The Wastewander rogues in Tanaris now using stealth while patroling at the zone.
- The NPC Sunfury Nethermancer now correctly attack target, casting spells.

##### CHANGES PROPOSED:

-  Updated SAI /DB
-  

###### ISSUES ADDRESSED:

Closes 

##### TESTS PERFORMED:

The sql is imported without issues. Tested in game, the all listed creatures are correct now.

##### HOW TO TEST THE CHANGES:

1. .go creature 105757  ( Attack that NPC from the ground )
2. .go creature 123494  ( Try to interact with that NPC )
3. .go creature 23465   ( These Rogues must be in Stealth )
4. .go creature 71960   ( Attack Sunfury Nethermancer - They must attack you, and cast some spells instead of just standing doing nothing).

I'll include some pictures / videos : 
Rogues before:
Rogues

Rogues after:
RoguesFixed

Geirrvif before : 
Geirrvif

Geirrvif after:
Geirrvifcorrect

Emerald before:
https://youtu.be/3dgFQwNk1hw

Emerald after: 
https://youtu.be/7B_Bqbh180k

Sunfury before: 
https://youtu.be/tmr8ybfByFQ

Sunfury after:
https://youtu.be/i6Mn20MtVRg

##### KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

##### Target branch(es):

Master

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1553120573757251500.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1553120573757251500');
+
+-- NPC "Emerald Skytalon" entering/leaving combat and doesn't fly down to attack player (Fix)
+REPLACE INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `VerifiedBuild`) VALUES (105757, 27244, 571, 0, 0, 1, 1, 24453, 0, 2801.95, 7.25871, 4.26567, 4.96257, 300, 0, 0, 9940, 0, 0, 0, 0, 0, 0);
+
+-- NPC "Geirrvif" flying too high. The players cant interact with NPC (Fix)
+REPLACE INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `VerifiedBuild`) VALUES (123494, 31135, 571, 0, 0, 1, 1, 27074, 0, 8216.64, 3516.06, 624.996, 3.57661, 300, 0, 0, 12600, 0, 0, 0, 0, 0, 0);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1553120573757251500.sql]: can you please use `DELETE` and `INSERT` instead of `REPLACE` ?

Comment (wizzymore) [reply data/sql/updates/pending_db_world/rev_1553120573757251500.sql]: Here, change with this:
```sql
DELETE FROM `creature` WHERE `guid` = 105757;
INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES(105757, 27244, 571, 0, 0, 1, 1, 24453, 0, 2801.95, 7.25871, 4.26567, 4.96257, 300, 0, 0, 9940, 0, 0, 0, 0, 0, '', 0);

DELETE FROM `creature` WHERE `guid`=123494;
INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES(123494, 31135, 571, 0, 0, 1, 1, 27074, 0, 8216.64, 3516.06, 624.996, 3.57661, 300, 0, 0, 12600, 0, 0, 0, 0, 0, '', 0);

```

Comment (poszer) [reply data/sql/updates/pending_db_world/rev_1553120573757251500.sql]: Sure @FrancescoBorzi , thanks @wizzymore

**Comment by comix1988** [comment]
I tested this PR and everything works.
good job @poszer 

tested on Ubuntu 18.04.2 LTS

**on behalf of @MaelstromServer**

**Comment by poszer** [comment]
> I tested this PR and everything works.
> good job @poszer
> 
> tested on Ubuntu 18.04.2 LTS
> 
> **on behalf of @MaelstromServer**

Thank you so much for testing @comix1988

**Comment by Viste** [comment]
@poszer Thank you!

**Pull request merged**